### PR TITLE
Updated media library configuration info

### PR DESCRIPTION
**Pull request by amitkhanal**
**Summary**
Updated documentation for Cloudinary Media Library integration.

**Motivation**
Documentation need updating around syntax and approach to integrating Cloudinary Media Library

### CODE REVIEW

File: website/content/docs/cloudinary.md [hunk 1]
```diff
@@ -30,55 +30,101 @@ media_library:
 The following options are specific to the Netlify CMS integration for Cloudinary:
 
 * **`output_filename_only`**: _(default: `false`)_\
-  By default, the value provided for a selected image is a complete URL for the asset on Cloudinary's CDN. Setting `output_filename_only` to `true` will instead produce just the filename (e.g. `image.jpg`).
+  By default, the value provided for a selected image is a complete URL for the asset on Cloudinary's CDN. Setting `output_filename_only` to `true` will instead produce just the filename (e.g. `image.jpg`). This should be `true` if you will be directly embedding cloudinary transformation urls in page tempaltes. Refer to [Embedding Cloudinary url in templates] ()
```
Comment (erquhart) [inline website/content/docs/cloudinary.md]: End of this sentence needs to be completed (missing url, space between brackets and parens, needs a period).

File: website/content/docs/cloudinary.md [hunk 2]
```diff
@@ -30,55 +30,101 @@ media_library:
...
 * **`use_transformations`**: _(default: `true`)_\
   If `true`, uses derived url when available (the url will have image transformation segments included). Has no effect if `output_filename_only` is set to `true`.
 * **`use_secure_url`**: _(default: `true`)_\
   Controls whether an `http` or `https` URL is provided. Has no effect if `output_filename_only` is set to `true`.
 
 ## Cloudinary configuration options
 
-The Cloudinary media library integration can be configured in two ways: globally and per field. Global options will be overridden by field options. All options are listed in Cloudinary's [media library documentation](https://cloudinary.com/documentation/media_library_widget#3_set_the_configuration_options), but only the following options are available or recommended for the Netlify CMS integration:
```
Comment (erquhart) [inline website/content/docs/cloudinary.md]: We should indicate the global vs. per field options.

File: website/content/docs/cloudinary.md [hunk 3]
```diff
@@ -30,55 +30,101 @@ media_library:
...
+fields: # The fields each document in this collection have
+      - label: 'Cover Image'
+        name: 'image'
+        widget: 'image'
+        required: false
+        tagname: ''
+        media_library:
+          config:
+            default_transformations:
+              -   - fetch_format: auto
```
Comment (erquhart) [inline website/content/docs/cloudinary.md]: Is the nested array intentional here?

**Comment by erquhart** [comment]
Great improvements, thanks @amitkhanal! Just a few questions in the review comments.

File: website/content/docs/cloudinary.md (hunk 3 above)
Comment (amitkhanal) [reply website/content/docs/cloudinary.md]: Yes, this is intentional. The default_transformation type should be `Object[][]`.

File: website/content/docs/cloudinary.md (hunk 1 above)
Comment (amitkhanal) [reply website/content/docs/cloudinary.md]: Fixed

File: website/content/docs/cloudinary.md (hunk 2 above)
Comment (amitkhanal) [reply website/content/docs/cloudinary.md]: Added reference links for global and field options - under section `Transforming images via Media Library`

**Comment by amitkhanal** [comment]
@erquhart I've pushed new changes. Please review.

**Comment by erquhart** [comment]
Thanks for the updates!

**Pull request merged**